pypeit_cache_github_data keck_deimos misses some calibration data

The nodes in my high performance computing environment do not have outside access so I ran `pypeit_cache_github_data keck_deimos` to cache everything Pypeit needs. However, `run_pypeit` still attempts to download some stuff:
```
[ERROR]   :: utils.py 573 fetch_remote_file() - Error downloading keck_deimos_600ZD.fits: <urlopen error [Errno -2] Name or service not known>
             URL attempted: https://raw.githubusercontent.com/pypeit/PypeIt/1.11.0/pypeit/data//arc_lines/reid_arxiv/keck_deimos_600ZD.fits
             If the error relates to the server not being found,
             check your internet connection.  If the remote server
             name has changed, please contact the PypeIt development
             team.
```

And this appears to occur for every grating.

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -4,6 +4,7 @@
 
 - (Hotfix) Fixed bug that allowed science frames to be assigned to multiple
   instrument configurations
+- (Hotfix) Fixed typo related to GitHub download for offline processing
 
 1.11.0 (21 Oct 2022)
 --------------------

--- a/pypeit/data/utils.py
+++ b/pypeit/data/utils.py
@@ -654,7 +654,7 @@ def _build_remote_url(f_name, f_type, remote_host=""):
         # Hard-wire the URL based on PypeIt Version
         data_url = (
             "https://raw.githubusercontent.com/pypeit/PypeIt/"
-            f"{'develop' if '.dev' in __version__ else __version__}/pypeit/data/"
+            f"{'develop' if '.dev' in __version__ else __version__}/pypeit/data"
         )
         return f"{data_url}/{f_type}/{f_name}", None
 

